docs/global-queue.md

```diff
@@ -0,0 +1,36 @@
+Global queue defines a deterministic total order of transactions across all `KeyRange`s using quorum offsets and epochs.
```

Comment:
Header and article missing.

In fact, I like your idea of one term per one markdown file so much that I'd suggest to go further and have a glossary, right in `docs/README.md`!